### ISSUE TITLE
Ideas to promote this project

### ISSUE THREAD

**Issue by hhkaos** [hhkaos/arcgis_websocket_server#4]
* Create several stream services to subscribe to #GISDay, another for hashtags used from Esri accounts.
* Publish these services in now.sh or similar
* Create public items in ArcGIS Online with a link to this project
* Add a contact form
* Publish an article in GeoNet
* etc.

Search for other open sources in real time:
* Fiware
* Cars on the move?
* Weather?

> And add them to [the documentation](https://github.com/hhkaos/arcgis_websocket_server#documentation)

**Comment by hhkaos** [comment]
Make a story map showing some real time capabilities on the platform

**Comment by keithfraley** [comment]
would love to help promote this project, would you have time to talk about your vision?

**Comment by hhkaos** [comment]
Sure @keitofraley, give us a couple of weeks. @ntkog and me are trying to finish a few improvements before start promoting this project. I'll get back to you as soon as have finished it. Thx!

**Comment by keithfraley** [comment]
is there any update on this project?

**Comment by ntkog** [comment]
Yep!  We didn't merge yet, but in this [repo](https://github.com/ntkog/demo_custom_streamserver) you can check it out ( Let me know if the instructions are easy to follow) 
If you want to see the changes or the implementation, check this [repo](https://github.com/ntkog/arcgis-websockets-streamserver)
